content/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018.md

```diff
@@ -0,0 +1,80 @@
+the four robots in the arena could move any token, this required the robots to
+detect which tokens were where and to react to changes.
+
+The matches were broken into two main sections. First was a league stage with
+matches spaced throughout the weekend and interspersed with "tinker time" which
+both allowed the teams to test and improve their robots as well as gauge the
+strengths of their opponents. The time between rounds proved invaluable to many
+of the teams and we saw the robots improve greatly over the course of the
+weekend.
+
+Once the league was complete, the results from the league seeded a knockout
+stage. While every robot entered the knockouts, it all came down to the final
+match between _[Noo Noo][CLY-robot-image]_ from [Collyer's][collyers] team
+_SCC_, _[Out of the blue][RGS-robot-image]_ from [RGS Guildford][rgs-guidlford],
+_[Brenda][BPV2-robot-image]_ from [Barton Pevril][barton-pevril]'s second team
+and [the robot][BRK-robot-image] from team _Bodgenhurst_ from
+[Brokenhurst][brokenhurst].
+
+[barton-pevril]: https://www.barton-peveril.ac.uk/
+[brokenhurst]: https://www.brock.ac.uk/
+[collyers]: http://www.collyers.ac.uk/
+[rgs-guidlford]: https://www.rgs-guildford.co.uk/
+
+[BPV2-robot-image]: /img/robots-2018/BPV2.jpg
+[BRK-robot-image]: /img/robots-2018/BRK.jpg
+[CLY-robot-image]: /img/robots-2018/CLY.jpg
+[RGS-robot-image]: /img/robots-2018/RGS.jpg
+
+The final promised to be interesting as each of the robots had fairly different
+approaches. _Out of the Blue_, _Brenda_ and _Bodgenhurst_'s robot each attempted
+to capture tokens in their open front halves by driving up to tokens with their
+arms open, before closing them around the tokens.
+
+_Out of the Blue_ extended this strategy by first driving through the tokens in
+their own zone with their arms closed, which pushed some of the undesirably
+tokens out of their zone before they collected their own tokens. This had given
+the robot a strong position after the league matches, by ranking second they
+went straight through to the semi-finals.
+
+In the end however it was _Noo Noo_'s great use of its [Mecanum
+wheels][mecanum-wheels] and long grabber arm to drive _sideways_ through their
+own zone at the start of the match, sweeping the undesirable tokens out of the
+zone, which granted their victory.
+
+[mecanum-wheels]: https://en.wikipedia.org/wiki/Mecanum_wheel
+
+### Prizes
+
+| Prize                 | Team                            |
+| --------------------- | ------------------------------- |
+| 1<sup>st</sup> Place  | Collyer's — SCC                 |
+| 2<sup>nd</sup> Place  | RGS Guildford — Out Of The Blue |
+| 3<sup>rd</sup> Place  | Brokenhurst — Bodgenhurst       |
+| Most Ambitious        | RGS Guildford — Out Of The Blue |
+| Best Looking          | Barton Pevril — Orion           |
+
+### Livestream
+
+This year's competition was streamed live on Facebook, thanks to the
```

Comment:
My opinion is that this sentence would feel less stilted if the first and last commas were removed, but that's up to you.